### ISSUE TITLE
[Bug Report] v-select aria-readonly attr is false on input element but should be true

### ISSUE THREAD

**Issue by chrmorqmatic** [vuetifyjs/vuetify#12982]
### Environment
**Vuetify Version:** 2.4.2
**Vue Version:** 2.6.10
**Browsers:** Chrome 87.0.4280.141
**OS:** Windows 10

### Steps to reproduce
check html result input element

### Expected Behavior
`<input id="input-3" readonly="readonly" type="text" aria-readonly="true" autocomplete="off">`

### Actual Behavior
`<input id="input-3" readonly="readonly" type="text" aria-readonly="false" autocomplete="off">`

### Reproduction Link
<a href="https://codepen.io/chrmorqmatic/pen/jOMdaLa" target="_blank">https://codepen.io/chrmorqmatic/pen/jOMdaLa</a>

### Other comments
This should be consistent, both value should be the same, in the v-select sourcecode, readonly is always true on the input element but the aria-readonly comes from this.isReadonly ~~that doesn't exists and thus is aways false.~~

**Comment by KaelWD** [comment]
Does this cause problems in screen readers? 

> readonly is always true on the input element

So you can't type in it, it's a select component

> aria-readonly comes from this.isReadonly that doesn't exists

It's inherited from `v-text-field`, set the readonly prop and you'll see it change.

**Comment by chrmorqmatic** [comment]
> Does this cause problems in screen readers?

Yes, we received the report from a customer auditing our application for WCAG accessibility, here it is:

This <input> has a mismatch between the value of its readonly and aria-readonly attributes
When ARIA attributes (state, roles, and properties) are not used correctly assistive technology may not correctly function as expected. Valid markup should always be used.
Developers must use valid markup and follow the rules of the ARIA specification.
 
> It's inherited from `v-text-field`, set the readonly prop and you'll see it change.

Yes, this workaround works, thanks

**Comment by KaelWD** [comment]
> assistive technology **may not** correctly function as expected

Emphasis mine. If this hasn't actually been confirmed in a screen reader it's just hot air. There's nothing invalid about using different `readonly` and `aria-readonly` values if that's the desired behaviour. 

> adding readonly prop makes the v-select readonly

This was to refute your claim that "this.isReadonly doesn't exists"

**Comment by chrmorqmatic** [comment]
>If this hasn't actually been confirmed in a screen reader it's just hot air. 

We'll check that and keep you informed

>This was to refute your claim that "this.isReadonly doesn't exists"

Sorry I thought you were trying to help by giving a workaround. I've changed the initial bug report removing that wrong information

**Comment by chrmorqmatic** [comment]
> We'll check the screen readers and keep you informed

We have tested this issue with NVDA and chrome

**Comment by Ta-nvi** [comment]
Hello, I am having the same issue but want the `aria-readonly=true`so that it matches the `readonly` tag. However, since `<input>` is rendered as html, I am unable to set it using `v-select` component. Any workarounds/fixes found yet?
